skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,23 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        from scipy.spatial.distance import pdist
```

Comment:
Unfortunately now I'm hung up on a circular import at [line 6 in `convex_hull.py`](https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/convex_hull.py#L6).